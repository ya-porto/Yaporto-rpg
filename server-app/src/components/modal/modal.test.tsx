import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import {mount, configure} from 'enzyme';
import renderer from 'react-test-renderer';
import {Modal} from './index';

configure({adapter: new Adapter()});
describe('Modal test', () => {
	const content = 'Modal content';
	const template = <Modal show>{content}</Modal>;
	it('Snapshot', () => {
		const tree = renderer.create(template).toJSON();
		expect(tree).toMatchSnapshot();
	});
	it('Content', () => {
		const comp = mount(template);

		expect(comp.text()).toMatch(content);
	});
});
