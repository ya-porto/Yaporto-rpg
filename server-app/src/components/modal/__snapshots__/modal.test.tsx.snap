// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal test Snapshot 1`] = `
<div
  className="modal "
>
  <div
    className="modal-content create-chat d-flex flex-column justify-space-between align-center pa-6 undefined"
  >
    Modal content
  </div>
</div>
`;
